common/Unit.cpp

```diff
@@ -537,6 +537,8 @@ void UnitBase::exitTest(TestResult result, const std::string& reason)
 
     // Notify inheritors.
     onExitTest(result, reason);
+
+    SocketPoll::wakeupWorld();
```

Comment:
Both `UnitKit::onExitTest()` and `UnitWSD::onExitTest()` already do call `SocketPoll::wakeupWorld()`. I just double-checked.

Reply:
Fair cop - dropped that.